lib/owd/request.rb

```diff
@@ -17,7 +18,9 @@ def perform
       request.body = xml
       request["Content-Type"] = "text/xml"
 
-      parse_response(http.request(request))
+      Timeout::timeout(timeout_seconds) {
```

Comment:
@ktransier I was unable to test that the Net::ReadTimeout was working - several people seemed to say it doesn't. Any ideas?